i18n/en.json

```diff
@@ -1184,8 +1184,10 @@
   "admin.service.corsExposedHeadersTitle": "CORS Exposed Headers:",
   "admin.service.corsHeadersEx": "X-My-Header",
   "admin.service.corsTitle": "Enable cross-origin requests from:",
+  "admin.service.createBotTitle": "",
```

Comment:
Need to copy the "_default" text from the code above into this file for translation.

Reply:
Do I do this manually? I ran `make i18n-extract` to get that output.

Reply:
@lieut-data or @jespino would you know why the output for this text is empty when using the `make i18n-extract`?

Reply:
I haven't yet used the `i18n-extract` target in the web app myself, but I know @bradjcoughlin has run into an issue with the automatic extraction, so I suspect it's not yet supported. @jespino is the authority though.

Reply:
Ok, that is the expected behavior, i18n-extract try to extract the I'd and the default string when is possible, but sometimes is not possible through static analysis so is responsibility of the developer to fill that text with the default string.